api/src/main/java/com/javadiscord/jdi/internal/api/DiscordRequestDispatcher.java

```diff
@@ -21,6 +21,7 @@ public class DiscordRequestDispatcher implements Runnable {
     private static final Logger LOGGER = LogManager.getLogger();
     private final BlockingQueue<DiscordRequestBuilder> queue;
     private final String botToken;
+    private boolean appIsActive;
```

Comment:
Maybe rename to just `active` or similar

Reply:
`running` is general convention but anything that tells a develop it's intent is good enough